test/segments/dir.spec

```diff
@@ -72,6 +72,26 @@ function testTruncationFromRightWorks() {
   unset POWERLEVEL9K_SHORTEN_STRATEGY
 }
 
+function testTruncationFromRightWithEmptyDelimiter() {
+  POWERLEVEL9K_SHORTEN_DIR_LENGTH=2
```

Comment:
Yep. I agree. Don't know why I didn't do this in the first place... To reduce merge conflicts, I would want to start that in the async branch.